x-pack/plugins/enterprise_search/public/applications/applications/components/playground/page_template.tsx

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
adding this so that playground and search applications can have different breadcrumbs.